dali/ft/filecopy.cpp

```diff
@@ -2646,7 +2650,7 @@ void FileSprayer::pushParts()
 
     performTransfer();
 }
-
+#ifdef _USE_AZURE
```

Comment:
picky: The ApiCopyStatus and IAPICopyClient should be before the #ifdef.  (E.g. Once we also support AWS)